src/command_modules/azure-cli-sql/azure/cli/command_modules/sql/commands.py

```diff
@@ -3,6 +3,28 @@
 # Licensed under the MIT License. See License.txt in the project root for license information.
 # --------------------------------------------------------------------------------------------
 
-from azure.cli.core.commands import cli_command
+from ._util import get_sql_servers_operation, ServiceGroup, create_service_adapter
 
-cli_command(__name__, 'sql server list', 'azure.cli.command_modules.sql.custom#list_sql_server')
+
+server_operations = create_service_adapter('azure.mgmt.sql.operations.servers_operations',
+                                           'ServersOperations')
+
+with ServiceGroup(__name__, get_sql_servers_operation, server_operations) as s:
+    with s.group('sql server') as c:
+        c.command('create', 'create_or_update')
+        c.command('delete', 'delete')
+        c.command('show', 'get_by_resource_group')
+        c.command('usage', 'list_usages')
```

Comment:
Since 'usage' is a noun, show this be 'show-usage' to conform with the "command names always have a verb" convention. For example, the command to show usage for a storage account is `az storage account show-usage`